components/calendar/src/any_calendar.rs

```diff
@@ -620,8 +620,8 @@ impl AnyCalendarKind {
             "indian" => AnyCalendarKind::Indian,
             "coptic" => AnyCalendarKind::Coptic,
             "iso" => AnyCalendarKind::Iso,
-            "ethiopic" => AnyCalendarKind::Ethiopic,
-            "ethioaa" => AnyCalendarKind::Ethioaa,
+            "ethiopic" => AnyCalendarKind::Ethiopian,
+            "ethioaa" => AnyCalendarKind::EthiopianAmeteAlem,
```

Comment:
Praise: `EthiopianAmeteAlem` is much better than `Ethioaa` as a type name. Like `JapaneseExtended` was better than `Japanext`.